### ISSUE TITLE
Organizational chart Accessibility

### ISSUE THREAD

**Issue by fguru06** [highcharts/highcharts#17798]
#### Expected behaviour
should be accessible through a tab or keyboard.

#### Actual behaviour
It's not selecting any point of the organizational chart

#### Affected browser(s)
All browsers

https://www.highcharts.com/docs/chart-and-series-types/organization-chart

**Comment by oysteinmoseng** [comment]
We are hoping to do some user testing to determine the best way to navigate org charts. For now, a basic workaround would be to set `mode: 'serialize'` and `skipNullPoints: false`: https://jsfiddle.net/g1camr0w/

**Comment by fguru06** [comment]
Thanks for your quick response @oysteinmoseng , this works perfectly.
